_posts/app/2000-01-01-force-https.md

```diff
@@ -64,14 +64,17 @@ Network](https://developer.mozilla.org/fr/docs/S%C3%A9curit%C3%A9/HTTP_Strict_Tr
 
 ## Disclaimers
 
-* Once the feature is enabled and that some users received the HSTS header,
-  they will **always** keep using HTTPS to reach your application, make sure it
-  works correctly, even after if the feature is disabled.
+* Make sure your application works correctly using HTTPS before enabling the
+  **Force HTTPS** feature. Once enabled, the application won't be reachable without HTTPS.
 
-* This feature adds a small lock-in to the platform as it depends of our
-  Infrastructure, if that is something you want to avoid, you can completely
-  implement it at your application levels. Libraries exist for most languages
-  and framework to create the exact same behavior. Here are a few examples:
+* Once the feature is enabled and that some users have received the HSTS header,
+  they will **always** keep using HTTPS to reach your application. Hance, make sure HTTPS
+  access keep working correctly, even if the feature disabled.
+
+* This feature adds a small lock-in to the platform as it depends of Scalingo's
```

Comment:
depends of -> depends on: I think that `depends` must always be followed by `on`.